### ISSUE TITLE
QUIC - PHP - setting of connection referencing variables

### ISSUE THREAD

**Issue by krombel** [mholt/caddy#1141]
### 1. What version of Caddy are you running (`caddy -version`)?
Caddy 0.9.2

#### 1.1 Because i think that it also may be a PHP related issue:
I am using php 5.6.24-0+deb8u1

### 2. What are you trying to do?
Do authentication with PHP based on 
```_SERVER["REMOTE_HOST"]``` and ```_SERVER["REMOTE_PORT"]	```

### 3. What is your entire Caddyfile?
```text
example.com {
    tls webmaster@example.com
    root   /var/www/html

    fastcgi / unix:/var/run/php5-fpm-www.sock php {
           env PATH /bin
    }
    rewrite {
        r ^/index.php/.*$
        to /index.php?{query}
    }

    status 403 /forbidden
    # .htacces / data / config / ... shouldn't be accessible from outside
    rewrite {
        r  ^/(?:\.htaccess|README|.git)
        to /forbidden
    }
    header / Strict-Transport-Security "15768000"

}
```

### 4. How did you run Caddy (give the full command and describe the execution environment)?
caddy -quic -log stdout -agree=true

### 5. What did you expect to see?
That ```_SERVER["REMOTE_HOST"]``` and ```_SERVER["REMOTE_PORT"] ``` are containing values

### 6. What did you see instead (give full error messages and/or log)?
```phpinfo()``` tells me that both values are containing ```"no value"```

### 7. How can someone who is starting from scratch reproduce this behavior as minimally as possible?
Start caddy with quic enabled and open a php-file containing ```phpinfo();``` via a quic-enabled connection.

**Comment by elcore** [comment]
Hello @krombel,

I am not able to reproduce your issue!

Caddy 0.9.3 + PHP 7.0.8 [7.0.8-0ubuntu0.16.04.2] (Ubuntu 16.04)

**Comment by krombel** [comment]
seems to be fixed in 0.9.3
